setup.py

```diff
@@ -26,7 +26,7 @@
             'torch == 1.10.1',
             'torchvision == 0.11.2',
             'torchaudio == 0.10.1',
-            'pytorch-lightning',
+            'pytorch-lightning==1.6.5',  # 1.7 requires protobuf version incompatible with tensorflow/tensorboard. Otherwise works.
```

Comment:
not blocking - in theory we could isolate the pytorch and tensorflow installs to their own python virtual envs, thus avoiding this conflict and allowing pytorch lightning to resolve as high as possible

Unless of course the tensorboard dependency i used in pytorch....then please ignore me.

Reply:
> we could isolate the pytorch and tensorflow installs to their own python virtual envs

Good idea but I don't know how to do that. 

In practice there is very little advantage to being on the absolute latest package and I suspect the tensorboard team will update their deps shortly.